sci-biology/hmmer/Changelog

```diff
@@ -0,0 +1,7 @@
+# ChangeLog for sci-biology/squiz
+# Copyright 1999-2016 Gentoo Foundation; Distributed under the GPL v2
+# $Id$
+
+  22 Jul 2016 Bertrand Néron <freeh4cker@gmail.com> hmmer-3.1_beta2,
+	update SRC_URI
+
```

Comment:
Ok, I did not know that.
I will drop it

On Sat, Jul 23, 2016 at 1:12 PM, Marius Brehler <notifications@github.com>
wrote:

> In sci-biology/hmmer/Changelog
> <https://github.com/gentoo-science/sci/pull/651#discussion_r71971156>:
>
> > @@ -0,0 +1,7 @@
> > +# ChangeLog for sci-biology/squiz
> > +# Copyright 1999-2016 Gentoo Foundation; Distributed under the GPL v2
> > +# $Id$
> > +
> > +  22 Jul 2016 Bertrand Néron <freeh4cker@gmail.com> hmmer-3.1_beta2,
> > +	update SRC_URI
> > +
>
> With portage now using git, changelogs were dropped. Same for the science
> overlay, since we have the git history.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gentoo-science/sci/pull/651/files/dfa3f177ba681a327737f1b1d0805f78071441a9#r71971156>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALDS58O73lasphEXraG0KEU18SM6BSlIks5qYfcDgaJpZM4JS5xq>
> .
>